csv/unstable_stringify.ts

```diff
@@ -280,55 +308,15 @@ export type StringifyOptions = {
  * );
  * ```
  *
- * @example Give an array of string arrays and specify columns with renaming
- * ```ts
- * import { stringify } from "@std/csv/unstable-stringify";
- * import { assertEquals } from "@std/assert/equals";
- *
- * const data = [
- *   ["Rick", 70],
- *   ["Morty", 14],
- * ];
- *
- * const columns = [
- *   { prop: 0, header: "name" },
- *   { prop: 1, header: "age" },
- * ];
- *
- * assertEquals(
- *   stringify(data, { columns }),
- *  `name,age\r\nRick,70\r\nMorty,14\r\n`,
- * );
- * ```
- *
- * @example Emit TSV (tab-separated values) with `separator: "\t"`
- * ```ts
- * import { stringify } from "@std/csv/unstable-stringify";
- * import { assertEquals } from "@std/assert/equals";
- *
- * const data = [
- *   ["Rick", 70],
- *   ["Morty", 14],
- * ];
- *
- * assertEquals(stringify(data, { separator: "\t" }), `Rick\t70\r\nMorty\t14\r\n`);
- * ```
- *
- * @example Prepend a byte-order mark with `bom: true`
- * ```ts
- * import { stringify } from "@std/csv/unstable-stringify";
- * import { assertEquals } from "@std/assert/equals";
- *
- * const data = [["Rick", 70]];
- *
- * assertEquals(stringify(data, { bom: true }), "\ufeffRick,70\r\n");
- * ```
- *
  * @param data The source data to stringify. It's an array of items which are
  * plain objects or arrays.
  * @param options Options for the stringification.
  * @returns A CSV string.
  */
+export function stringify(
+  data: readonly ObjectItem[],
+  options?: ObjectStringifyOptions,
```

Comment:
This only expose ObjectStringifyOptions to the users. (ArrayStringifyOptions is only used in implementation signature, and not visible from the user)

Reply:
I think we also need:

```
export function stringify(
  data: readonly ArrayItem[],
  options?: ArrayStringifyOptions,
): string;
```

BTW If we overload a function, the doc linter requires each overload (except implementation signature) having its own jsdoc.

Reply:
done